corehq/apps/hqwebapp/templates/hqwebapp/crispy/next_action_form.html

```diff
@@ -7,6 +7,7 @@
     hx-target="#{{ container_id }}"
     hx-disabled-elt="find button"
     hq-hx-action="{{ next_action }}"
+    enctype="multipart/form-data"
```

Comment:
i think this should be fine, but since this is used by data cleaning, just make sure that those forms aren't affected :)

Reply:
Yes, tested, data cleaning is not affected!